src/pages/_plugins/jekyll-filters.rb

```diff
@@ -45,10 +45,6 @@ def first_letters(value)
       value.to_s.split.map(&:chr).join
     end
 
-    def divide(value, number)
```

Comment:
it's jekyll filter, not sass mizin, you cannot remove it, becaouse for example this line will be not work: 

```
{% assign all-days = days | minus: start-day | plus: 1 | divide: 7.0 | ceil | times: 7 %}
```